letsdrill.rb

```diff
@@ -1,12 +1,25 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+  grade = ''
+  case integer
+  when 90..100
+    grade = 'A'
+  when 75..89
+    grade = 'B'
+  when 65..74
+    grade = 'C'
+  when 50..64
+    grade = 'D'
+  else
+    grade = 'F'
+  end
 
+  return grade
```

Comment:
ruby functions will return the last evaluated expression so explicit returns aren't used often except in the case of guard clauses.

example where someone might use an explicit return.

```
def is_tall?(person)
  return false if person.short?
  true
end
```